tests/docker/bin/cluster_up.sh

```diff
@@ -37,11 +37,14 @@ docker-compose -f $PY_DOCKER_COMPOSE_FILE up -d
 
 echo "Setting throttle for throttle test..."
 docker-compose -f $PY_DOCKER_COMPOSE_FILE exec kafka sh -c "
-        /usr/bin/kafka-configs  --zookeeper zookeeper:2181 \
+        /usr/bin/kafka-configs  --bootstrap-server kafka:9092 \
                 --alter --add-config 'producer_byte_rate=1,consumer_byte_rate=1,request_percentage=001' \
                 --entity-name throttled_client --entity-type clients"
 
+echo "Awaiting schema-registry..."
```

Comment:
Since the name is passed to await_http, why not just let await_http print the awaiting thingie?